apps/web/src/common/components/mfa/components/EmailFoldingInfo.vue

```diff
@@ -61,7 +76,7 @@ const handleClickSendEmailButton = async () => {
             {{ $t('COMMON.MFA_MODAL.COLLAPSE_DESC') }}
             <p-text-button class="send-code-button"
                            style-type="highlight"
-                           :disabled="(props.isDisabledModal || props.isReSyncModal) ? !storeState.email : !props.isSentCode"
+                           :disabled="(props.isDisabledModal || props.isReSyncModal) ? !storeState.email || isCooldown : !props.isSentCode || isCooldown"
```

Comment:
suggestion: It's hard to grasp the conditions at a glance😅 How about using a `computed` variable instead?